packages/react-swipeable-views/package.json

```
@@ -28,7 +28,7 @@
     "warning": "^4.0.1"
   },
   "peerDependencies": {
-    "react": "^15.3.0 || ^16.0.0"
+    "react": ">=15.3.0"
```

Peer dependency warnings are nonblocking. Best to be safe

```suggestion
    "react": "^15.3.0 || ^16.0.0 || ^17.0.0"
```